cli/lib/exec/spawn.js

```diff
@@ -234,7 +235,8 @@ module.exports = {
       }
 
       return spawn(overrides)
-      .then((code) => {
+      .then((code, signal) => {
```

Comment:
wait - here did you mean `.then(({code, signal}) =>`?

Reply:
ah, no, but `signal` will always be null here... let me fix this

Reply:
k, all better now